tests/utils.py

```diff
@@ -86,6 +87,23 @@ def random_gamma_dist_integer(inclusive_max, max_tries=100):
     raise RuntimeError(f"Max tries exceeded while generating a random integer in range [1, {inclusive_max}]")
 
 
+def generate_asset_tree(root_external_id: str, depth: int, children_per_node: int, current_depth=1):
```

Comment:
Let's keep this asset-specific test helper in `test_api/test_assets`.

Reply:
...but it exists in two different places, with slightly different implementation (`name` is missing in one of them)

Reply:
right, but if we won't be needing it in `test_unit/test_api/test_assets` anymore, then it will only be used in `test_integration/test_api/test_assets`